Makefile

```diff
@@ -23,3 +23,23 @@ export GOPROXY=https://proxy.golang.org
 .PHONY: docker-build
 docker-build: build
 
+# Helper functions to improve local testing experience
```

Comment:
For reviewers: Don't know how you feel about having these, I just added them to speed up local testing quite a bit - can remove if preferred :)